### DISCUSSION TITLE
Is it necessary to automatically add "--jar xx.jar" when adding jar of resources?

### DISCUSSION THREAD

**Discussion by choucmei** [apache/dolphinscheduler#8524] [Q&A]
<img width="706" alt="image" src="https://user-images.githubusercontent.com/32946731/155490066-44e2133a-235e-415b-9911-c0fee8c25308.png">

**Comment by caishunfeng** [comment]
Do you mean that add default UI tip here?  
Does it need to add `--jar xx.jar` every time for using `Option Parameters`?

**Reply by choucmei** [reply]
> Do you mean that add default UI tip here?
> Does it need to add `--jar xx.jar` every time for using `Option Parameters`?

step1: add jars to resources,
step2: add jar to spark env like '--jars xx.jar'.
i mean the second step could be automatically?

**Reply by caishunfeng** [reply]
OK, I get yout point now. It sounds cool.
